src/views/Weather.vue

```diff
@@ -1,30 +1,53 @@
             <button type="submit">Get Weather</button>
         </form>
         <div v-if="result.temp">
             <!-- Weather Display -->
-            <p>Condition: {{ weather_main }}, {{ weather_description }}</p> <!-- Display weather description -->
-            <p>Temperature: {{ convert_kelvin_to_fahrenheit(result.temp) }}°F ({{ convert_kelvin_to_celsius(result.temp) }}°C)</p> <!-- Displays temp in F and C -->
-            <p>Feels Like: {{ convert_kelvin_to_fahrenheit(result.feels_like) }}°F ({{ convert_kelvin_to_celsius(result.feels_like) }}°C)</p> <!-- Displays feels like in F and C -->
-            <p>Humidity: {{ result.humidity }}%</p> <!-- Displays air humidity -->
-            <p>Pressure: {{ result.pressure }} hPa</p> <!-- Displays air pressure in hPa -->
-            <p>Wind Speed: {{ convert_mps_to_mph(wind_speed) }} mph ({{ wind_speed }} m/s)</p> <!-- Displays wind speed in mph and m/s -->
-            <p>Wind Direction: {{ wind_direction }}°</p> <!-- Display wind direction -->
-            <p>Cloudiness: {{ clouds_percent }}%</p> <!-- Display cloud coverage -->
+            <img v-if="weather_icon" :src="`https://openweathermap.org/img/wn/${weather_icon}@2x.png`" alt="Weather Icon" /> <!-- Display weather icon -->
+            <p class="location-info"><b>Coordinates:</b> {{ latitude }}, {{ longitude }}</p>
+            <p class="location-info"><b>Condition:</b> {{ weather_main }}, {{ weather_description }}</p> <!-- Display weather description -->
+            <p class="location-info"><b>Temperature:</b> {{ convert_kelvin_to_fahrenheit(result.temp) }}°F ({{ convert_kelvin_to_celsius(result.temp) }}°C)</p> <!-- Displays temp in F and C -->
+            <p class="location-info"><b>Feels Like:</b> {{ convert_kelvin_to_fahrenheit(result.feels_like) }}°F ({{ convert_kelvin_to_celsius(result.feels_like) }}°C)</p> <!-- Displays feels like in F and C -->
+            <p class="location-info"><b>Humidity:</b> {{ result.humidity }}%</p> <!-- Displays air humidity -->
+            <p class="location-info"><b>Pressure:</b> {{ result.pressure }} hPa</p> <!-- Displays air pressure in hPa -->
+            <p class="location-info"><b>Wind Speed:</b> {{ convert_mps_to_mph(wind_speed) }} mph ({{ wind_speed }} m/s)</p> <!-- Displays wind speed in mph and m/s -->
+            <p class="location-info"><b>Wind Direction:</b> {{ wind_direction }}° ({{ getWindDirectionLetter(wind_direction) }})</p> <!-- Display wind direction in degrees and letter direction --> <!-- Display wind direction -->
+            <p class="location-info"><b>Cloudiness:</b> {{ clouds_percent }}%</p> <!-- Display cloud coverage -->
+            <p></p>
             <p>Weather provided by OpenWeather</p>
         </div>
         <div v-else>
+            <p></p>
             <p>No weather data found for {{ city }}</p> <!-- Error Handling -->
         </div>
     </div>
+    <!-- MapView component to display the map -->
+    <MapView ref="mapView" />
```

Comment:
We'll talk about storing the destination's coordinates in the store for use with this in todays meeting